include/nds/dma.h

```diff
@@ -66,18 +66,20 @@ extern "C" {
 #define DMA_BUSY            BIT(31)
 #define DMA_IRQ_REQ         BIT(30)
 
-#define DMA_START_NOW       0
-#define DMA_START_CARD      (5 << 27)
+#define DMA_START_NOW       0         ///< Start DMA immediately
 
 #ifdef ARM7
-#    define DMA_START_VBL   BIT(27)
+#    define DMA_START_VBL   (1 << 28) ///< Auto-start DMA on vertical blank
+#    define DMA_START_CARD  (2 << 28) ///< Auto-start DMA on Slot-1 card transfer
 #endif
 
 #ifdef ARM9
-#    define DMA_START_HBL   BIT(28)
-#    define DMA_START_VBL   BIT(27)
-#    define DMA_START_FIFO  (7 << 27)
-#    define DMA_DISP_FIFO   (4 << 27)
+#    define DMA_START_VBL   (1 << 27) ///< Auto-start DMA on vertical blank
+#    define DMA_START_HBL   (2 << 27) ///< Auto-start DMA on horizontal blank
```

Comment:
Maybe add the note here that there is in GBATEK that mentions "paused during V-Blank". It may be useful to be used as a quick reference.